client/utils/index.js

```diff
@@ -212,3 +213,23 @@ export const getExportLanguage = ( language, storedLanguage ) => {
 	// When modal is dismissed use the default language locale.
 	return language !== '' ? language : defaultLanguage;
 };
+
+/**
+ * Returns the language options for CSV exports language selector.
+ *
+ * @return {Array} Language options.
+ */
+export const exportLanguageOptions = () => {
```

Comment:
The name of this method makes it appear that we are exporting language options here. WDYT about `getExportLanguageOptions`?